Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -317,8 +317,17 @@ final class IfconfigExprTests: XCTestCase {
       #endif
       """,
       diagnostics: [
-        DiagnosticSpec(message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion")
-      ]
+        DiagnosticSpec(
+          message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion",
+          fixIts: ["insert '_version'", "insert '_underlyingVersion'"]
```

Comment:
Sorry for not noticing this in the first review. But the Fix-It actually replaces `unknown` by `_version`, so we shouldn’t be adding a Fix-It that says `insert`. `exchangeTokens` should be able to handle the generation of the diagnostic for you.

Reply:
I just added a different Fix-It message(`replace 'unknown' with '_version' and ':'`) for this case.